int32 latitude
int32 longtitude
uint8 transmission_state
uint16 speed
uint16 heading
int16 latitude_acceleration
int16 longtitude_acceleration
uint8 vehicle_class
string events
uint8 response_type
uint8 light_use

string id
uint16 sec_mark
int32 elevation
uint16 accuracy_semi_major
uint16 accuracy_semi_minor
uint16 accuracy_orientation
uint8 confidence_position
uint8 confidence_elevation
int16 angle
int16 vert_acceleration
int16 yaw_acceleration
uint8 brake_padel
string wheel_brakes
uint8 traction
uint8 abs
uint8 scs
uint8 brake_boost
uint8 aux_brakes
uint16 vehicle_width
uint16 vehicle_lenth
uint8 vehicle_height
uint8 vehicle_fuel_type
string lights
uint8 siren_use

















